src/stream_wrap.cc

```diff
@@ -204,7 +198,7 @@ void StreamWrap::OnReadImpl(ssize_t nread,
     return;
   }
 
-  char* base = static_cast<char*>(node::Realloc(buf->base, nread));
+  char* base = node::UncheckedRealloc(buf->base, nread);
```

Comment:
I'd make this a checked realloc.  It can't really fail because it shrinks but I doubt the extra check hurts.